### PR TITLE
bpo-31418: Fix an assertion failure in case of an exception with a bad __module__ attribute

### PR DESCRIPTION
**Pull request by orenmn**
https://bugs.python.org/issue31418

### CODE REVIEW

**Comment by orenmn** [comment]
feel free to remove the NEWS.d item if you don't think it is necessary.

File: Misc/NEWS.d/next/Core and Builtins/2017-09-13-13-03-52.bpo-31418.rS-FlC.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Fix an assertion failure in case of an exception with a bad ``__module__``
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Core and Builtins/2017-09-13-13-03-52.bpo-31418.rS-FlC.rst]: "in PyErr_WriteUnraisable()"

File: Misc/NEWS.d/next/Core and Builtins/2017-09-13-13-03-52.bpo-31418.rS-FlC.rst [hunk 2]
```diff
@@ -1,2 +1,2 @@
-Fix an assertion failure in case of an exception with a bad ``__module__``
-attribute. Patch by Oren Milman.
+Fix an assertion failure in `c.PyErr_WriteUnraisable()` in case of an
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Core and Builtins/2017-09-13-13-03-52.bpo-31418.rS-FlC.rst]: What is `c.PyErr_WriteUnraisable()`?

Comment (orenmn) [reply Misc/NEWS.d/next/Core and Builtins/2017-09-13-13-03-52.bpo-31418.rS-FlC.rst]: https://devguide.python.org/committing/#what-s-new-and-news-entries says:
`(single backticks) in reST can be used to refer to objects in the documentation.`
I thought this means that i should specify (inside single backticks) an object in the docs, according to how it appears in the docs (i.e. when pressing the `Permalink to this definition` button - https://docs.python.org/3.7/c-api/exceptions.html#c.PyErr_WriteUnraisable).

so it should just be `PyErr_WriteUnraisable()`?

Comment (serhiy-storchaka) [reply Misc/NEWS.d/next/Core and Builtins/2017-09-13-13-03-52.bpo-31418.rS-FlC.rst]: Yes, it should be without "c." prefix.

**Comment by orenmn** [comment]
sorry for the trouble, and thanks for your patience, Serhiy

**Review by serhiy-storchaka — APPROVED** [review]

**Pull request merged**